### ISSUE TITLE
Allow solid background color for ABKFeedNavigationBar

### ISSUE THREAD

**Issue by ldiqual** [Appboy/appboy-ios-sdk#68]
I'm trying to set a solid background color for ABKFeedNavigationBar. All I could come up with is this:

```
ABKFeedNavigationBar {
    background-tint-color: #3FCF7F;
    background-color: #3FCF7F;
}
```

![image](https://cloud.githubusercontent.com/assets/941453/15304020/cdb2107a-1b6f-11e6-9a8b-1e74fa03c87a.png)

Is there a way to make the navigation bar opaque with a solid green at the moment? Thank you :)

**Comment by Wenzhi** [comment]
Hi @ldiqual,

If you are using a non-navigation context, including generic, modal and popover contexts, you can make the navigation bar look solid by setting the translucent property:
```
ABKFeedViewControllerGenericContext *newsFeed = [[ABKFeedViewControllerGenericContext alloc] init];
[newsFeed.navigationBar setTranslucent:NO];
```

Please let us know if it doesn't work for you, or if you have any other questions.

Thanks,
Wenzhi

**Comment by ldiqual** [comment]
@Wenzhi I'm not initting the context myself, I'm assuming it just auto-creates itself whenever an inapp notification comes through. Is there a way to set this globally?

**Comment by Wenzhi** [comment]
@ldiqual, have you tried `[[UINavigationBar appearance] setTranslucent:NO] `?

**Comment by ldiqual** [comment]
@Wenzhi That would modify the appearance of all navigation bars, I want to apply it only to AppBoy's newsfeed.

**Comment by Wenzhi** [comment]
Hi @ldiqual,

Since the class for the feed navigation bar, `ABKThemableFeedNavigationBar`, isn't public, it’s not clear there’s a very clean, simple way to do this.  Instead of suggesting possibly fragile workarounds, we’d like to plan a more generic way of providing this functionality.  However, we don’t have a clear timeline for that now.  Any sense of how urgent and how high priority this is for you would be appreciated as we strategize enabling this functionality.

Please let us know any further comments/questions.

Thanks,
Wenzhi

**Comment by ldiqual** [comment]
@Wenzhi This is not urgent for us. Just let us know whenever it can be done :) Thanks for the followup!

**Comment by ldiqual** [comment]
@Wenzhi Any update on this?

**Comment by briancaw** [comment]
Hi @ldiqual 

We haven't been able to execute on this just yet, but with iOS 10 stuff slowing down we'll aim to prioritize it in the near future.  

Thanks very much for your patience,
Brian

**Comment by ldiqual** [comment]
@Wenzhi Bump on this. Still not urgent, but it seems that a quick fix from your side could be to make `ABKThemableFeedNavigationBar` public? Let me know.

**Comment by Wenzhi** [comment]
Hey @ldiqual,

Thanks for your suggestion and sorry for such a late reply! We just released SDK 2.26.0 and made the related classes public in the release. We also have the sample code [here](https://github.com/Appboy/appboy-ios-sdk/blob/master/Example/Stopwatch/AppDelegate.m#L76-L77) for how to set the News Feed navigation bar background color to be opaque.

Please let us know if it does __not__ work for you, or if you have any other questions or issues!

Thanks,
Wenzhi

**Comment by Wenzhi** [comment]
Hey @ldiqual,

I'm going to close this issue, assuming the latest SDK solved it. If you still see the issue happen or have any questions, please feel free to re-open this.

Thanks,
Wenzhi
